.eslintrc.js

```diff
@@ -2,98 +2,126 @@ module.exports = {
   root: true,
 
   // Make ESLint compatible with TypeScript
-  parser: '@typescript-eslint/parser',
```

Comment:
You've got the wrong prettier version - run `npm install` and re-run `eslint --fix .`